packages/gatsby/src/datastore/datastore.ts

```diff
@@ -2,47 +2,15 @@ import { IDataStore } from "./types"
 import { emitter } from "../redux"
 
 let dataStore: IDataStore
-let isLmdb = isLmdbStoreFlagSet()
 
 export function getDataStore(): IDataStore {
   if (!dataStore) {
-    if (isLmdb) {
-      const { setupLmdbStore } = require(`./lmdb/lmdb-datastore`)
-      dataStore = setupLmdbStore()
-    } else {
-      const { setupInMemoryStore } = require(`./in-memory/in-memory-datastore`)
```

Comment:
Didn't delete the `in-memoy` directory yet. I guess we don't want to keep it?

Reply:
We still use `runFastFiltersAndSort` in LMDB for the `runQuery` without indexes https://github.com/gatsbyjs/gatsby/blob/remove-locked-in/packages/gatsby/src/datastore/lmdb/lmdb-datastore.ts#L209-L218

Reply:
We need to keep it. Not only lmdb indexes is not enabled by default, but also lmdb indexes just can't handle certain filters as well as filter+sort combinations - https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby/src/datastore/lmdb/query README might have a bit more information about details, but TL/DR is that even LMDB indexes might need to fallback to using fast filters